contracts/tests/test_unruggable_memecoin.cairo

```diff
@@ -60,32 +82,61 @@ mod erc20_metadata {
         let name = 'UnruggableMemecoin';
         let symbol = 'UM';
         let initial_supply = 1000.into();
-        let contract_address = deploy_contract(owner, recipient, name, symbol, initial_supply);
+        let initial_holder_1 = contract_address_const::<44>();
+        let initial_holder_2 = contract_address_const::<45>();
+        let initial_holders = array![recipient, initial_holder_1, initial_holder_2].span();
+        let initial_holders_amounts = array![900.into(), 50.into(), 50.into()].span();
+        let contract_address =
+            match deploy_contract(
```

Comment:
could we have a test utility function to make it smoother and avoid duplication ?